.rubocop.yml

```diff
@@ -16,3 +16,6 @@ AllCops:
   <% `git status --ignored --porcelain`.lines.grep(/^!! /).each do |path| %>
     - <%= path.sub(/^!! /, '').sub(/\/$/, '/**/*').rstrip %>
   <% end %>
+
+Style/FrozenStringLiteralComment:
+  EnforcedStyle: never
```

Comment:
Here we are overriding the `runger_style` rule (rather than changing it upstream) because most projects that use `runger_style` aren't also going to use `freezolite` and so they should continue to use the `frozen_string_literal: true` comments.